resources/views/bread/edit-add.blade.php

```diff
@@ -47,7 +47,7 @@ class="form-edit-add"
 
                             <!-- Adding / Editing -->
                             @php
-                                $dataTypeRows = $dataType->{(isset($dataTypeContent->id) ? 'editRows' : 'addRows' )};
+                                $dataTypeRows = $dataType->{($dataTypeContent->getKey() ? 'editRows' : 'addRows' )};
```

Comment:
This applies to a number of changes here.  `->getKey()` returns the key value, not a boolean value